### ISSUE TITLE
Version completion of Mark Fixed

### ISSUE THREAD

**Issue by Phyxion** [exceptionless/Exceptionless.UI#196]
If you type 1.0, it's completed to 1.0.0 automatically. But this is not a good completion if you want to write 1.0.1. The completion of 1.0 to 1.0.0 should only occur after pressing **Mark Fixed** button.

**Comment by niemyjski** [comment]
Yeah that is kind of annoying, it's an angular thing. I could debounce it longer of which I thought I had it set to a half second.

**Comment by ejsmith** [comment]
Why are we even changing it before it's sent to the server? Seems like a very weird thing to be changing someone's input as they are typing.

**Comment by Phyxion** [comment]
For 1.0.0.0 to 1.0.0-0 it's useful as 1.0.0.0 isn't accepted, but I think these should only be modified after clicking on send. Or at least only for full versions.
